src/core/server/elasticsearch/api_types.ts

```diff
@@ -143,6 +143,7 @@ import {
   TasksCancelParams,
   TasksGetParams,
   TasksListParams,
+  SnapshotCleanupRepositoryParams,
```

Comment:
1 *nit*: Other type imports are ordered by name.
2 `elasticsearch` doesn't export this type.
```
kibana: src/core/server/elasticsearch/api_types.ts(146,3): error TS2724:
Module '"../../../../node_modules/@types/elasticsearch"' has no exported member 'SnapshotCleanupRepositoryParams'. Did you mean 'SnapshotCreateRepositoryParams'?
```

Reply:
This has been fixed by Alison's commit.

Reply:
👍 I don't see any changes related to the platform team